lib/features/events/presentation/bloc/event/event_event.dart

```diff
@@ -1,3 +1,61 @@
+import 'package:calentre/config/enums/weekdays.dart';
+import 'package:calentre/features/events/data/models/calentre_event.dart';
+
 class CalentreEventEvent {}
 
 class ProceedToSetAvailabilityEvent extends CalentreEventEvent {}
+
+class UpdateCalentreEventDetailsEvent extends CalentreEventEvent {
+  String? amount;
+  String? duration;
+  String? eventDescription;
+  String? eventLink;
+  String? eventName;
+  String? eventType;
+  String? isMultiple;
+  String? platformType;
+  Days? days;
+
+  UpdateCalentreEventDetailsEvent(
+      {this.amount,
+      this.duration,
+      this.eventDescription,
+      this.eventLink,
+      this.eventName,
+      this.eventType,
+      this.isMultiple,
+      this.platformType,
+      this.days});
+}
+
+class UpdateDayScheduleEvent extends CalentreEventEvent {
```

Comment:
This is mostly because of `Bloc` naming convention. Generally accepted to append `Event` and `State` to the names of classes for Bloc events and states respectively.